forest/cli/src/cli/info_cmd.rs

```diff
@@ -0,0 +1,152 @@
+// Copyright 2019-2023 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use std::time::Instant;
+
+use clap::Subcommand;
+use colored::*;
+use forest_blocks::tipset_keys_json::TipsetKeysJson;
+use forest_rpc_client::{
+    chain_get_name, chain_get_tipset, chain_head, start_time, wallet_default_address,
+};
+use forest_shim::econ::TokenAmount;
+use forest_utils::io::parser::{format_balance_string, FormattingMode};
+use time::OffsetDateTime;
+
+use super::Config;
+use crate::cli::handle_rpc_err;
+
+#[derive(Debug, Subcommand)]
+pub enum InfoCommand {
+    Show,
+}
+
+#[derive(Debug, strum_macros::Display)]
+enum SyncStatus {
+    Ok,
+    Slow,
+    Behind,
+}
+
+pub struct NodeStatusInfo {
+    /// timestamp of how far behind the node is with respect to syncing to head
+    behind: u64,
+    /// Chain health is defined as the amount of blocks in last finality divided
+    /// by healthy amount of blocks in a finality (900 epochs * 5 blocks per
+    /// tipset).
```

Comment:
Let's rewrite this to something simpler. What we're really measuring is how close we are to having an average of 5 blocks per tipset in the last couple of hours. You could write that the number of blocks per tipset is non-deterministic but averaging at 5 is considered healthy.

Reply:
Also, we might want to copy `fvm_shared::BLOCKS_PER_EPOCH` over to Forest and use it. It defines the desired number of blocks per epoch.

Reply:
Updated health definition, and using BLOCKS_PER_EPOCH